core/rawdb/accessors_indexes.go

```diff
@@ -434,6 +434,7 @@ func DeleteBlockLvPointers(db ethdb.KeyValueStore, blocks common.Range[uint64],
 // FilterMapsRange is a storage representation of the block range covered by the
 // filter maps structure and the corresponting log value index range.
 type FilterMapsRange struct {
+	Version                      uint32
```

Comment:
This will invalidate the previously stored entry and reindex the stuff in next launch, just want to mention

Reply:
This is intentional as also mentioned in the commit title, see also the description of https://github.com/ethereum/go-ethereum/pull/31642. The index has to be regenerated as it might be corrupted in a some cases.